modules/api/src/main/java/org/apache/ignite/sql/ColumnMetadata.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.sql;
+
+/**
+ * Interface that provides methods for accessing column metadata.
+ */
+public interface ColumnMetadata {
```

Comment:
Should we expose the default value here as well?

Reply:
What default value do you mean?
This column is not the same as Table column.
What default value should C1 column metadata return for the query "SELECT count(*) as C1 FROM ..." ?

Reply:
Agree, does not make sense, thanks.